gps/cmd_unix.go

```diff
@@ -78,7 +78,7 @@ func (c cmd) CombinedOutput() ([]byte, error) {
 	}()
 
 	if err := c.Cmd.Wait(); err != nil {
-		return nil, err
+		return b.Bytes(), err
 	}
 	return b.Bytes(), nil
```

Comment:
Obviously this is correct and the smallest change, but IMHO the resulting state is confusing and repetitive (The kind of thing which in the future would make me wonder how it got that way, and if something was lost in the process). What about this instead?
```
err := c.Cmd.Wait()
return b.Bytes(), err
```

Reply:
ah yeah, this is a reasonable simplification. i'll modify it.